TLDR.md

```diff
@@ -0,0 +1,18 @@
+
+
+# Introduction
+
+Hey! This is Purple Bubble. We're a crew of teens on a mission to make messaging epic again. Zero tracking, spying, or data selling. Everything is end-to-end encrypted, signed, and decentralized. All open source.
+
+> **Wanna chat?**
+>
+> We mostly communicate through the [Purple Bubble Slack](https://join.slack.com/t/thepurplebubble/signup), but you can also email us (team@purplebubble.org) or open a Github issue.
+ 
+
+## Protocol
+
+Protocol v1.0.0 Jasper has multiple REST API servers in a mesh network. Clients pick two servers. One is used for transmission (TX) and the other is used for reception (RX). For faster messaging users can switch to Transparent Mode, which uses an encrypted STUN server to establish a WebRTC connection. Messages are signed by users' keys.
+
+## App
+
+We currently plan to have a desktop and mobile app. You'll be able to sign in using a key file that holds your public and private key, and your friends can add you by importing a contact file that holds your public key, name, and preferred profile picture.
```

Comment:
Let's not get into feature specifics here.

Reply:
please elaborate

Reply:
imo it's fine